### DISCUSSION TITLE
Context: lang en power bi

### DISCUSSION THREAD

**Discussion by amoradel** [tmijail/Odoo-Power-BI-Connector#19] [Q&A] [answered]
Hola Mijail,

Muchas gracias por tu apoyo,

Una consulta ¿Como se puede colocar el context lang `'context': {'lang': "es_GT"}` en el serach_read de power Bi?

En python se hace de esta manera, intente hacerlo siguiendo la sintaxys de power bi, pero no me lo acepta
![image](https://github.com/tmijail/Odoo-Power-BI-Connector/assets/10636860/9dc6b384-7b7a-4043-a71d-4f634e0964d1)

**Comment by tmijail** [comment, marked as answer]
Es similar, pero hay que tener cuidado con las llaves y los corchetes.
Este es un ejemplo de una query de uno de mis reportes en la que uso `context`:

```
= search_read(
    "product.product", 
    {}, [
    context=[active_test=false],
    fields = {
        "default_code",
        "display_name",
        "active",
        "categ_id",
        "qty_available",
        "replenishment_cost",
        "list_price",
        "product_tmpl_id"
}])
```

**Reply by amoradel** [reply]
Muchas gracias Mijail funciona excelente,

Una consulta extra 😅 ¿Conoces si puedo obtener fechas por medio del context con una zona horaria especifica? Lo que sucede es que yo obtengo los datos de la BD con la zona horaria 0 UTC, pero yo me encuentro en la -6 UTC.

**Reply by tmijail** [reply]
*Me parece* que se puede con el contexto `tz`, pero lo que suelo hacer es traer los valores en UTC y en PowerBI restarle a la columna `#duration(0, 6, 0, 0)`.

**Reply by amoradel** [reply]
Esto lo hago en la transformación de datos de Power Query Mijail?

![image](https://github.com/tmijail/Odoo-Power-BI-Connector/assets/10636860/7a67a6a7-640d-4190-a8d8-9871bbb24716)

**Reply by amoradel** [reply]
Ya lo pude realizar de esta manera.

![image](https://github.com/tmijail/Odoo-Power-BI-Connector/assets/10636860/2bd32cd4-bcad-44ae-9d29-796c35212041)

Muchas gracias siempre por tu ayuda Mijail.
